test/test.js

```diff
@@ -601,8 +609,7 @@ suite('Methods', function() {
     })
   })
 
-  // TODO: Waiting to verify behavior
```

Comment:
Yeah, right now the Chrome implementation also throws an exception on GET/HEAD with body. There is a [suggestion to change the spec to allow](https://github.com/whatwg/fetch/issues/83) bodies for GET requests, but until the spec actually changes, I think these tests are goo.d